src/context/VideoContext.js

```diff
@@ -4,6 +4,9 @@ import { videoReducer } from "../reducers";
 
 const initialState = {
   videos: [],
+  filters: {
+    category: "All",
+  },
   loading: false,
   error: null,
```

Comment:
I didn't understand why you give for every context, loading false and error.

Reply:
It's just for initial fetching of data.